public/app/features/dashboard/components/DashboardSettings/schema.ts

```diff
@@ -0,0 +1,190 @@
+// NOTE!
+// This should be loaded from:
+// http://localhost:3000/api/schema/dashboard.json
```

Comment:
Why do we have this in addition to just the JSON files that are served from the backend?